tests/conformance_tests.rs

```diff
@@ -79,17 +79,90 @@ mod implementation {
+        "ion-tests/conformance/system_macros/add_macros.ion",
+        // Error: found operation name with non-symbol type: sexp
+        "ion-tests/conformance/ion_encoding/module/load_symtab.ion",
+        "ion-tests/conformance/ion_encoding/module/symtab.ion",
+        // Error: Too few arguments.
+        "ion-tests/conformance/ion_encoding/module/macro/cardinality/invoke_cardinality_tl.ion",
+        // Error: "Invalid macro name:"
+        "ion-tests/conformance/ion_encoding/module/macro/trivial/signature.ion",
+        // Error: ExpectedSignal: No such macro: NoSuchMacro
+        "ion-tests/conformance/ion_encoding/module/macro/trivial/invoke_tl.ion",
+        // Error: ExpectedSIgnal: invalid argument
+        "ion-tests/conformance/system_macros/add_symbols.ion",
+        "ion-tests/conformance/system_macros/set_macros.ion",
+        "ion-tests/conformance/system_macros/set_symbols.ion",
+        // Error: Decoding Error: macro none signature has 0 parameters(s), e-expression had an
+        // extra argument.
+        "ion-tests/conformance/system_macros/default.ion",
+        // System macro delta not yet implemented
+        "ion-tests/conformance/system_macros/delta.ion",
+        // System macro make_decimal not yet implemented
+        "ion-tests/conformance/system_macros/make_decimal.ion",
+        // System macro repeat not yet implemented
+        "ion-tests/conformance/system_macros/repeat.ion",
+        // System macro parse_ion not yet implemented
+        "ion-tests/conformance/system_macros/parse_ion.ion",
+        // System macro sum not yet implemented
+        "ion-tests/conformance/system_macros/sum.ion",
+        // System macro make_timestamp not yet implemented
+        "ion-tests/conformance/system_macros/make_timestamp.ion",
+        // Expected Signal: invalid macro definition
+        "ion-tests/conformance/tdl/expression_groups.ion",
+        // Expected Signal Errors and e-expression transcription
+        "ion-tests/conformance/tdl/if_none.ion",
+        "ion-tests/conformance/tdl/if_some.ion",
+        "ion-tests/conformance/tdl/if_multi.ion",
+        "ion-tests/conformance/tdl/if_single.ion",
+    ];
+
+    #[test_resources("ion-tests/conformance/core/*.ion")]
+    #[test_resources("ion-tests/conformance/data_model/*.ion")]
+    #[test_resources("ion-tests/conformance/eexp/*.ion")]
+    #[test_resources("ion-tests/conformance/eexp/binary/*.ion")]
+    #[test_resources("ion-tests/conformance/ion_encoding/*.ion")]
+    #[test_resources("ion-tests/conformance/ion_encoding/module/*.ion")]
+    #[test_resources("ion-tests/conformance/ion_encoding/module/macro/cardinality/*.ion")]
+    #[test_resources("ion-tests/conformance/ion_encoding/module/macro/template/*.ion")]
+    #[test_resources("ion-tests/conformance/ion_encoding/module/macro/trivial/*.ion")]
+    #[test_resources("ion-tests/conformance/system_macros/*.ion")]
+    #[test_resources("ion-tests/conformance/tdl/*")]
```

Comment:
Does this macro support `**` syntax for nesting at any depth? (Or maybe you're enumerating the subdirectories to leave something out?)